<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="SubjectsTab" name="subjectsTab">
       <attribute name="title">
        <string>Matières</string>
       </attribute>
      </widget>
      <widget class="TeachersTab" name="teachersTab">
       <attribute name="title">
        <string>Enseignants</string>
       </attribute>
      </widget>
      <widget class="OptionsTab" name="optionsTab">
       <attribute name="title">
        <string>Options</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="computation">
       <attribute name="title">
        <string>Calcul</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nombre de groupes :</string>
            </property>
            <property name="buddy">
             <cstring>numberOfGroupsSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="numberOfGroupsSpinBox">
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="numberOfWeeksSpinBox">
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Nombre de semaines :</string>
            </property>
            <property name="buddy">
             <cstring>numberOfWeeksSpinBox</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Exporter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startButton">
            <property name="text">
             <string>Démarrer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="text">
             <string>Arrêter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SubjectsTab</class>
   <extends>QWidget</extends>
   <header>SubjectsTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TeachersTab</class>
   <extends>QWidget</extends>
   <header>TeachersTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OptionsTab</class>
   <extends>QWidget</extends>
   <header>OptionsTab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>numberOfGroupsSpinBox</tabstop>
  <tabstop>numberOfWeeksSpinBox</tabstop>
  <tabstop>table</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>stopButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
